pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,28 @@ def test_isnotnull(self):
                             'B': [True, False, True, True, False]})
         tm.assert_frame_equal(res.to_dense(), exp)
 
+    @pytest.mark.parameterize('index', [None, list('abc')])
```

Comment:
Not nice at all! Doesn't work! Without arguments' default values (`...=None`), all I'm getting is:

> TypeError: test_from_to_scipy() missing 3 required positional arguments: 'index', 'columns', and 'fill_value'

(With default values (`...=None`), those are the only values ever tested with, i.e. no param matrix.

https://paste.debian.net/917666/

Avoiding to read the whole of pytest docs, any clue as to where's the wrong I'm doing?